apps/prairielearn/src/sync/fromDisk/assessments.ts

```diff
@@ -141,7 +141,7 @@ function getParamsForAssessment(
             autoPoints: alternative.autoPoints ?? question.autoPoints ?? null,
             manualPoints: alternative.manualPoints ?? question.manualPoints ?? null,
             forceMaxPoints: alternative.forceMaxPoints ?? question.forceMaxPoints,
-            triesPerVariant: alternative.triesPerVariant ?? question.triesPerVariant,
+            triesPerVariant: alternative.triesPerVariant ?? question.triesPerVariant, // XXX
```

Comment:
This is why I think this is a bug: `alternative.triesPerVariant` now has a default of `1`, so it will never fallback to the question's triesPerVariant.